### ISSUE TITLE
Failed to resolve entry for package "groq-builder".

### ISSUE THREAD

**Issue by ringods** [FormidableLabs/groqd#247]
When trying `groq-builder` v0.1.3, I get this in my [astro](https://astro.build/) setup:

```
[commonjs--resolver] Failed to resolve entry for package "groq-builder". The package may have incorrect main/module/exports specified in its package.json.
```

My `package.json`:

```
{
  "name": "personal-website",
  "private": true,
  "version": "0.0.1",
  "scripts": {
    "dev": "astro dev",
    "start": "astro dev",
    "build": "astro check && astro build",
    "preview": "astro preview",
    "astro": "astro"
  },
  "dependencies": {
    "@astrojs/check": "0.3.1",
    "@astrojs/tailwind": "5.0.2",
    "@sanity/astro": "2.1.4",
    "@sanity/client": "6.9.1",
    "@sanity/image-url": "1.0.2",
    "astro": "3.6.4",
    "astro-portabletext": "0.9.6",
    "groq-builder": "0.1.3",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "sanity": "3.20.2",
    "tailwindcss": "3.3.5",
    "typescript": "5.3.2"
  },
  "devDependencies": {
    "daisyui": "^4.4.17"
  }
}
```

Strange thing is that in my VSCode editor, Typescript reports everything ok. No squigly red lines anywhere.

**Comment by ringods** [comment]
@scottrippey the `groq-builder` package contains this line:

https://github.com/FormidableLabs/groqd/blob/f35225efaf394ba1a4671da71dacc3dcdcf1bf0a/packages/groq-builder/package.json#L21

But the compiled package doesn't contain the file `dist/index.mjs`.

**Comment by scottrippey** [comment]
Thank you for reporting this!  Should be fixed by #248 !
